tests/pytests/test_resp3.py

```diff
@@ -208,11 +179,10 @@ def test_profile(env):
     }
     env.expect('FT.PROFILE', 'idx1', 'SEARCH', 'QUERY', '*', "FORMAT", "STRING").equal(exp)
 
+@skip(cluster=False, redis_less_than="7.0.0")
 def test_coord_profile():
     env = Env(protocol=3)
-    if should_skip(env):
-        env.skip()
-    if not env.isCluster() or env.shardsCount != 3:
+    if env.shardsCount != 3:
```

Comment:
We should add this also to the skip decorator (not mandatory on this PR).

Reply:
generalized the test instead